tests/vm_test.go

```diff
@@ -792,11 +792,12 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				Expect(err).ToNot(HaveOccurred())
 
 				By("Ensuring the VirtualMachineInstance is migrated")
-				Eventually(func() bool {
+				Eventually(func(g Gomega) {
 					vmi, err := virtClient.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
-					Expect(err).ToNot(HaveOccurred())
-					return vmi.Status.MigrationState != nil && vmi.Status.MigrationState.Completed
-				}, 240*time.Second, 1*time.Second).Should(BeTrue())
+					g.Expect(err).ToNot(HaveOccurred())
+					g.Expect(vmi.Status.MigrationState).ToNot(BeNil(), "vmi.Status.MigrationState should not be nil")
```

Comment:
nit: How about using BeEmpty instead of BeNil?

Reply:
Not in this case. BeEmpty id for collections like slices or maps. This field is a pointer to a struct type. We must check it's not nil before continuing to the next line.